app/helpers/admin/dashboard_helper.rb

```diff
@@ -36,4 +36,13 @@ def relevant_account_timestamp(account)
 
     content_tag(:time, l(timestamp), class: 'time-ago', datetime: timestamp.iso8601, title: l(timestamp))
   end
+
+  def emergency_mode_notice
+    reason = EmergencyMode.reason
+    return if reason.blank?
```

Comment:
You do have a `EmergencyMode.enabled?` method, so perhaps it'd be better to use that for the early return? e.g.,
```suggestion
    return unless EmergencyMode.enabled?
    
    reason = EmergencyMode.reason
```